pyhf/tensor/mxnet_backend.py

```diff
@@ -339,3 +350,31 @@ def log_prob(value, loc, scale):
                 scale, Number) else scale.log()
             return -((value - loc) ** 2) / (2 * variance) - log_scale - math.log(math.sqrt(2 * math.pi))
         return self.exp(log_prob(x, mu, sigma))
+
+    def normal_cdf(self, x, mu=0, sigma=1):
+        """
+        The cumulative distribution function for the Normal distribution
+
+        Example::
+
+            >>> pyhf.tensorlib.normal_cdf(0.8)
+
+            [0.7881446]
+            <NDArray 1 @cpu(0)>
+
+
+        Args:
+            x (`tensor` or `float`): The observed value of the random variable
+                                      to evaluate the CDF for
+            mu (`tensor` or `float`): The mean of the Normal distribution
+            sigma (`tensor` or `float`): The standard deviation of the Normal distribution
+
+        Returns:
+            MXNet NDArray: The CDF
+        """
+        # This is currently using SciPy stats until a better way can be found to
+        # do this in MXNet
+        x = self.astensor(x).asnumpy()
+        mu = self.astensor(mu).asnumpy()
+        sigma = self.astensor(sigma).asnumpy()
+        return self.astensor(norm.cdf(x, loc=mu, scale=sigma))
```

Comment:
I'm thinking whether or not this warrants a `log.warning`. Since this will definitely break autodifferentiability. We don't use this backend properly anyways, so adding this should poolute any logs.

Reply:
Removed in 745b87a

Reply:
oh I didn't mean in `astensor`, but for calls of `normal_cdf`

Reply:
Sorry, it was late and I misunderstood what you mean. You're saying if we should have the log warning on the comment I had to indicate that this is not fully MXNet, correct?

Reply:
Okay, should be actually fixed in 8b58fde now.